tox.ini

```
@@ -682,10 +682,11 @@ allowlist_externals =
   sh
 
 setenv =
+  ; override CORE_REPO via env variable when testing in forks
+  CORE_REPO=git+https://github.com/{env:CORE_REPO:open-telemetry/opentelemetry-python}.git@{env:CORE_REPO_SHA}
```

I would say to avoid those changes, maybe just use `GITHUB_REPOSITORY` runtime environment variable? Like
```suggestion
  CORE_REPO=git+https://github.com/{env:GITHUB_REPOSITORY:open-telemetry/opentelemetry-python}.git@{env:CORE_REPO_SHA}
```
At CI this will point to the current owner and repository name, so when running in your fork, it should match your check-spelling-sandbox/opentelemetry-python